src/azul/uuids.py

```diff
@@ -0,0 +1,77 @@
+import uuid
```

Comment:
The extraction of this could be a separate commit.

Reply:
I mean the extraction of the UUID validation functionality into a separate azul module.